spec/metrics/relation_spec.rb

```diff
@@ -24,20 +24,22 @@
   describe "#merge!" do
     it "merge multi values" do
       r1 = rel.where(id: [1, 2], dummy: 'qwe').time(:hour)
-      r2 = Influxer::Relation.new(DummyMetrics).where.not(user_id: 0).group(:user_id)
+      r2 = Influxer::Relation.new(DummyMetrics).where.not(user_id: 0).group(:user_id).order(user_id: :asc)
       r1.merge!(r2)
-      expect(r1.to_sql).to eq "select * from \"dummy\" group by time(1h),user_id where (id=1 or id=2) and (dummy='qwe') and (user_id<>0)"
+      expect(r1.to_sql)
+        .to eq "select * from \"dummy\" where (id=1 or id=2) and (dummy='qwe') and (user_id<>0) " \
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.